src/main/java/io/anserini/search/SearchMsmarco.java

```diff
@@ -127,9 +144,9 @@ public static void main(String[] args) throws Exception {
 
         SimpleSearcher.Result[] hits;
         if (retrieveArgs.fields.size() > 0) {
-          hits = searcher.searchFields(query, fields, retrieveArgs.hits);
+          hits = searcher.searchFields(queryGenerator, query, fields, retrieveArgs.hits);
```

Comment:
We could instead have `SimpleSearcher` be configured with a `QueryGenerator`, this could be nice and help avoid  having too many 'search' methods on `SimpleSearcher`.